ui/apps/akasha/src/components/post-page/post-page.tsx

```diff
@@ -281,6 +198,96 @@ const PostPage: React.FC<IPostPageProps & RootComponentProps> = props => {
     [entryData?.delisted, entryData?.moderated, isReported],
   );
 
+  const entryUi = () => {
```

Comment:
I would memoize this function or even extract it to a separate component

Reply:
I've already changed it to a separate component.  I had that in mind :) I will push the commit in a bit.